### ISSUE TITLE
ParsedDockerComposeFile fails if no parentFile (directory) is given in docker-compose file

### ISSUE THREAD

**Issue by haraldatbmw** [testcontainers/testcontainers-java#2718]
This code causes a NullPointerException because of missing directory info.

```java
File composeFile = new File("docker-compose-test.yml");
DockerComposeContainer<?> environment = new DockerComposeContainer<>(composeFile)
	.withExposedService("demo-app", 8080, Wait.forHttp("/demo").forPort(8080))
	.withTailChildContainers(false)
	.withLocalCompose(true)
	.withBuild(true)
	.withRemoveImages(RemoveImages.LOCAL);
```

This code works because of `./` as directory info.

```java
File composeFile = new File("./docker-compose-test.yml");
DockerComposeContainer<?> environment = new DockerComposeContainer<>(composeFile)
	.withExposedService("demo-app", 8080, Wait.forHttp("/demo").forPort(8080))
	.withTailChildContainers(false)
	.withLocalCompose(true)
	.withBuild(true)
	.withRemoveImages(RemoveImages.LOCAL);
```

**Exception**
```bash
java.lang.NullPointerException: null
        at org.testcontainers.containers.ParsedDockerComposeFile.findImageNamesInDockerfile(ParsedDockerComposeFile.java:124)
        at org.testcontainers.containers.ParsedDockerComposeFile.parseAndValidate(ParsedDockerComposeFile.java:91)
        at org.testcontainers.containers.ParsedDockerComposeFile.<init>(ParsedDockerComposeFile.java:43)
        at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
        at java.base/java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948)
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
        at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578)
        at org.testcontainers.containers.DockerComposeContainer.<init>(DockerComposeContainer.java:122)
        at org.testcontainers.containers.DockerComposeContainer.<init>(DockerComposeContainer.java:112)
        at org.testcontainers.containers.DockerComposeContainer.<init>(DockerComposeContainer.java:108)
```

**Comment by kiview** [comment]
This is a regression that has been introduced through changes in `DockerComposeContainer`.

**Comment by GooseMagnet** [comment]
@kiview I created a PR with the fix: https://github.com/testcontainers/testcontainers-java/pull/3866

**Comment by eddumelendez** [comment]
Fixed via #3866
